exercises/julia-1-a/runtests.jl

```diff
@@ -0,0 +1,214 @@
+using Test
+
+include("tictactoe.jl")
+
+@testset "Finished games where X won via colum victory" begin
+    boards = [
+        """
+        XOO
+        X  
+        X  
+        """,
```

Comment:
This is parsed as follows:

```julia
"XOO\nX  \nX  \n"
```

I chose the format to make my own life a bit easier, but it has no effect on the student.